rust/noosphere-cli/src/native/commands/serve/name_system/tasks.rs

```diff
@@ -0,0 +1,357 @@
+///! This task manager is not currently used, but a potential
```

Comment:
Need to give this some more thought and jam on how we can use this, but unused at the moment.